Code/GraphMol/Depictor/DepictUtils.cpp

```diff
@@ -193,6 +193,74 @@ int pickFirstRingToEmbed(const RDKit::ROMol &mol,
   return res;
 }
 
+RDKit::VECT_INT_VECT findCoreRings(const RDKit::VECT_INT_VECT &fusedRings,
+                                   RDKit::INT_VECT &coreRingsIds,
+                                   const RDKit::ROMol &mol) {
+  // simplify the fused rings to a set of core rings by iteratively removing
+  // rings that share only one or two consecutive atoms. These
+  // are trivial to embed after the core rings have been embedded and will make
+  // template matching more powerful since it will not be affected by the side
+  // rings
+
+  RDKit::INT_VECT removedRings;
```

Comment:
The pattern of using an INT_VECT to track things which have been modified is pretty inefficient and doesn't express intent particularly well.
Using a `boost::dynamic_bitset` instead (here: `boost::dynamic_bitset<> removedRings(fusedRings.size())` is generally faster and makes it clear that you're going to just be checking whether or not a particular index has been changed.